Makefile

```diff
@@ -410,13 +410,17 @@ multi-build: VERSION dev1-build
 # Build rules
 #
 
+<<<<<<< HEAD
```

Comment:
@rustaem ...

Reply:
> @rustaem ...

Sorry. Never should have make myself too accustomed to GUI process :(